### PR TITLE
proposal for WFCORE-5324, Ability to execute CLI script at runtime

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

**Review by honza-kasik — COMMENTED** [review]
Thanks @jfdenise !

File: bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc
Comment (jfdenise) [reply bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc:75]: Yes, the server starts automatically in admin-mode and reloads once the script has been executed.

File: bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc
Comment (jfdenise) [reply bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc:76]: Yes, the error shows the failing line and the reason (e.g.: command not found).

File: bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc
Comment (jfdenise) [reply bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc]: Fixed.

File: bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc
Comment (jfdenise) [reply bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc]: Yes, I added a note.

File: bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc
Comment (jfdenise) [reply bootable-jar/WFCORE_5324_CLI_script_exec_runtime.adoc:67]: The newLine are the one expected per operation system.Java handles it: "A line is considered to be terminated by any one of a line feed ('\n'), a carriage return ('\r'), or a carriage return followed immediately by a linefeed."

**Review by jmesnil — APPROVED** [review]

**Comment by jmesnil** [comment]
@honza-kasik sorry, I may have merged the PR a bit too fast. I should have wait for your approval (even though the feature PR has already been merged and released with WildFly Core 15.0.0.Final)

**Comment by honza-kasik** [comment]
@jmesnil No problem! The only thing that is missing is "testing by" chapter which can be filled after RFE receives rank. However I believe it will be tested by QE most probably.

**Pull request merged**
